README.md

```diff
@@ -1,3 +1,594 @@
+
+We have followed general OpenZeppelin Contracts guidelines: functions revert instead returning `false` on failure. This behavior is nonethelessconventional and does not conflict with the expectations of ERC20 applications.
+
+Additionally, an {Approval} event is emitted on calls to {transferFrom}. This allows applications to reconstruct the allowance for all accounts just by listening to said events. Other implementations of the EIP may not emit these events, as it isn't required by the specification.
+Finally, the non-standard {decreaseAllowance} and {increaseAllowance} functions have been added to mitigate the well-known issues around setting allowances. See {IERC20-approve}.
+
+#### Events
+None
+
+#### name
+Arguments:
+None
+
+Returns the name of the token.
+
+#### symbol
+Arguments:
+None
+
+Returns the symbol of the token.
+
+#### decimals
+Arguments:
+None
+
+Returns the number of decimals of the token.
+
+#### totalSupply
+Arguments:
+None
+
+Returns the total supply of the token.
+
+#### balanceOf
+Arguments:
+- `account` (address) - the address whose balance is being queried
+
+Returns the balance of account.
+
+#### transfer
+Arguments:
+- `to` (address) - the destination of the tokens being transferred
+- `amount` (uint256) - the number of tokens being transferred
+
+Transfers `amount` tokens from sender to address `to`.
+
+#### allowance
+Arguments:
+- `owner` (address) - the address that has allowed tokens to be used by another party
+- `spender` (address) - the address that has been approved to spend some of `owner`s tokens
+
+Returns the amount of tokens `owner` has approved `spender` to spend.
+
+#### approve
+Arguments:
+- `spender` (address) - the address being granted access to sender's tokens
+- `amount` (uint256) - the amount of tokens to which sender is granting `spender` access
+
+Allows `spender` to control `amount` of sender's tokens.
+
+#### transferFrom
+Arguments:
+- `from` (address) - the origin of the tokens being transferred
+- `to` (address) - the destination of the tokens being transferred
+- `amount` (uint256) - the number of tokens being transferred
+
+Transfers `amount` tokens from address `from` to address `to`.
+
+##### Security notes
+This method is limited by the amount of tokens `from` has approved the sender to spend on their behalf.
+
+#### increaseAllowance
+Arguments:
+- `spender` (address) - the address whose allowance is being changed
+- `addedValue` (uint256) - the additional amount `spender` is approved for
+
+Increases `spender`s approval from sender by `addedValue`.
+
+#### decreaseAllowance
+Arguments:
+- `spender` (address) - the address whose allowance is being changed
+- `subtracteValue` (uint256) - the amount by which `spender`s approval is being lessened
+
+Decreases `spender`s approval from sender by `subtractedValue`.
+
+
+
+#### The Deployment Tooling (/deploy)
```

Comment:
This section duplicates the information above. So can be removed ?
Also same comments around coverage apply